src/bci_build/package/git/README.md.j2

```diff
@@ -0,0 +1,36 @@
+# {{ image.title }}: Git application container
+{% include 'badges.j2' %}
+
+
+## Description
+
+Git is a distributed version control system that tracks
+versions of files. It is often used to control source code by programmers who
+are developing software collaboratively.
+
+Design goals of Git include speed, data integrity, and support for
+distributed, non-linear workflows — thousands of parallel branches running on
+different computers.
+
+
+## Usage
+
+This container provides the {{ image.publish_registry.vendor }} version of Git.
+
+Example of using git container:
```

Comment:
nit: shout it be Git or git?

Reply:
changed it to upper case, not sure it is better..